### DISCUSSION TITLE
Nano support for Kernel v5.10

### DISCUSSION THREAD

**Discussion by mglettig** [OE4T/meta-tegra#965] [General]
According to this [article ](https://forums.developer.nvidia.com/t/jetson-software-roadmap-for-2h-2021-and-2022/177724) the upcoming Jetpack release v5.0 will no longer support Jetson Nano. This is a pity as there is not yet a successor for the Nano. Jetson Nano would be stuck with kernel v4.9.
Is it worth to test the kernel coming with https://github.com/OE4T/meta-tegra/tree/wip-jetpack-5-dp on the Jetson Nano or is it expected to not work at all?
Kind regards,
Michael

**Comment by madisongh** [comment]
None of the device trees for the Nano (or other t210 platforms) are present in that kernel (or the rest of the L4T R34.1.0 BSP), so you probably wouldn't get very far with it.

If you don't care about CUDA and other JetPack features, you can run a mainline kernel on your Nano.  Searching around for "Jetson mainline kernel" should turn up some pages.  One of the NVIDIA dev forum posts I read recently also hinted at making it easier to run "other kernels" on the t210 and t186 platforms with future L4T R32.x releases, but I don't really know what that would mean.
